src/EventManagement.Services.Converto/ConvertoPdfRenderService.cs

```diff
@@ -0,0 +1,52 @@
+using losol.EventManagement.Services.Pdf;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace EventManagement.Services.Converto
+{
+    public class ConvertoPdfRenderService : IPdfRenderService
+    {
+        private readonly IHttpClientFactory httpClientFactory;
+        private readonly IOptions<ConvertoConfig> options;
+        private readonly ILogger<ConvertoPdfRenderService> logger;
+
+        public ConvertoPdfRenderService(
+            IHttpClientFactory httpClientFactory,
+            IOptions<ConvertoConfig> options,
+            ILogger<ConvertoPdfRenderService> logger)
+        {
+            this.httpClientFactory = httpClientFactory;
+            this.options = options;
+            this.logger = logger;
+        }
+
+        public async Task<Stream> RenderHtmlAsync(string html, PdfRenderOptions options)
+        {
+            var client = this.httpClientFactory.CreateClient();
+            var endpointUrl = this.options.Value.EndpointUrl;
+
```

Comment:
Do we need to authenticate here? Using values from config?

Reply:
Damn you are right my mistake, need to login first and add token here. Forgot about it

Reply:
@losolio should be done now. Had to add some tests to ensure it's working, can't live without them :)

Reply:
These tests are disabled now. The plan is to run some converto service with the well-kown URL, create user for testing there and then put it all into tests, and re-enable them.